src/clj_3df/compiler.cljc

```diff
@@ -87,8 +87,8 @@
                      :bool    boolean?))
 
 (s/def ::predicate '#{<= < > >= = not=})
-(s/def ::aggregation-fn '#{min max count median sum avg variance})
-(s/def ::function '#{truncate})
+(s/def ::aggregation-fn '#{min max count median sum avg})
```

Comment:
Leave `variance` in for now. I haven't even understood why it had to go and it's definitely part of the grammar :)